src/DistributedLock.AzureStorage/AzureStorageDistributedLock.cs

```diff
@@ -180,34 +186,19 @@ public async Task<bool> RenewLease(TimeSpan renewInterval)
 
             try
             {
-                await lease.Blob.RenewLeaseAsync(
-                    AccessCondition.GenerateLeaseCondition(lease.Token),
-                    renewRequestOptions,
-                    _storageContext).ConfigureAwait(false);
+                await lease.Blob.GetBlobLeaseClient(lease.Token).RenewAsync(renewRequestOptions);
```

Comment:
@Master-Oliveira  We should set Configure await false in all the internal async call we await

Reply:
Done, well spotted